covid_moonshot_ml/train_dd.py

```diff
@@ -90,15 +90,25 @@ def build_model_schnet(qm9=None, dg=False, qm9_target=10):
         # target=10 is free energy (eV)
         model_qm9, _ = SchNet.from_qm9_pretrained(qm9, qm9_dataset, qm9_target)
 
+        if remove_atomref:
+            atomref = None
+            ## Get rid of entries in state_dict that correspond to atomref
+            wts = {k: v for k,v in model_qm9.state_dict().items() \
+                if 'atomref' not in k}
+        else:
+            atomref = model_qm9.atomref.weight.detach().clone()
+            wts = model_qm9.state_dict()
+
+        model_params = (model_qm9.hidden_channels, model_qm9.num_filters,
+            model_qm9.num_interactions, model_qm9.num_gaussians,
+            model_qm9.cutoff, model_qm9.max_num_neighbors,model_qm9.readout,
+            model_qm9.dipole, model_qm9.mean, model_qm9.std, atomref)
+
         if dg:
-            model = SchNetBind(model_qm9.hidden_channels, model_qm9.num_filters,
-                model_qm9.num_interactions, model_qm9.num_gaussians,
-                model_qm9.cutoff, model_qm9.max_num_neighbors, model_qm9.readout,
-                model_qm9.dipole, model_qm9.mean, model_qm9.std,
-                model_qm9.atomref.weight.detach().clone())
-            model.load_state_dict(model_qm9.state_dict())
+            model = SchNetBind(*model_params)
         else:
-            model = model_qm9
+            model = SchNet(*model_params)
+        model.load_state_dict(wts)
 
     ## Set interatomic cutoff to 3.5A (default of 10) to make the graph smaller
     model.cutoff = 3.5
```

Comment:
Should we be experimenting with larger cutoffs too? Perhaps this could be set by the CLI?